protocol/avdl/keybase1/favorite.avdl

```diff
@@ -18,27 +18,29 @@ protocol favorite {
 }
 
   enum ConflictStateType {
-    AutomaticResolving_0,
-    ManualResolvingServerView_1,
+    ServerView_1,
     ManualResolvingLocalView_2
   }
 
-  record ConflictAutomaticResolving {
-    boolean isStuck;
-  }
-
-  record ConflictManualResolvingServerView {
+  record FolderServerView {
+    // Set to true if we're trying to automatically resolve conflict in this
+    // folder. Whether or not there's a local view version of this TLF doesn't
+    // affect this.
+    boolean resolvingConflict;
```

Comment:
I'm confused by this -- if this is supposed to be a "ServerView", how can it ever be resolving a conflict?  If we have a conflict locally and it's not stuck, then the user is not seeing the server view; they're seeing their local conflict view, and currently have no way of seeing the real server view until the conflict resolution is complete, or gets stuck and moved out of the way.  Maybe `FolderServerView` is just the wrong name for this structure?  Is the idea here that it's representing the current local state of the normal path to the TLF?  Maybe `FolderNormalView`?  `FolderNormalPathView`?

Reply:
Good point! That must have been why I did it with two separate types in the first place. Sigh. Sorry for not recalling this earlier. Normal view sounds reasonable; will update.